docs/proposals/compute-resource-metrics-api.md

```diff
@@ -115,13 +116,11 @@ only be available through the kubelet. The types of metrics are detailed
 Additionally, all endpoints (except root discovery endpoint) support the
 following optional query parameters:
 
-- `start` - start time to return metrics from; type json encoded
-  `time.Time`; since samples are retrieved at discrete intervals, the first
-  sample after the start time is the actual beginning.
-- `end` - end time to return metrics to; type json encoded `time.Time`
-- `step` - the time step between each stats sample; type int (seconds), default
```

Comment:
I think the parameters I've provided here are a superset of the prometheus instantaneous & range queries. If I understood correctly, an instantaneous query is equivalent to setting `maxSamples=1`, and a range query is equivalent to setting `sinceSeconds`? I'm still not sure how step should work, though I'm inclined to just have it drop samples that fall between steps.